### DISCUSSION TITLE
Welcome to HO Discussions!

### DISCUSSION THREAD

**Discussion by akasolace** [ho-dev/HattrickOrganizer#826] [General]
## 👋 Welcome!
  We’re using Discussions as a place to: 
  * Ask questions you’re wondering about.
  * Share ideas.

This place can be seen as an extension/replacement of HT forum.

**Comment by ygrepper** [comment]
Hello!
I have upgraded to HO! 4.1 and I am very greatful that there is still development going on for this tool. However my Team Rating is completely off track and I can't seem to get it adjusted... I have also tried the feedback plugin but get an error message in the end...
What am I doing wrong...? 

Thanks in advance for your help

BR Greppi

**Reply by akasolace** [reply]
> However my Team Rating is completely off track and I can't seem to get it adjusted... 

next version of HO! will have a completely revamped lineup tab. Meanwhile, please be sure you have prediction model set to new.
Also check all settings are correct (venue, ts, attitude ....) and lastly if you want to compare vs HT please select the minute 0 in the toggler (otherwise it defaults to 90 minute average).  

> I have also tried the feedback plugin but get an error message in the end

The feedback plugin is currently broken.

**Reply by ygrepper** [reply]
Thanks for your quick response. I did try all that, but not much improvement... Will wait until the next release and work with the old version in parallel for the player rating.
